nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/i18n/fi/LC_MESSAGES/ckanext-napote_theme.po

```diff
@@ -149,71 +176,73 @@ msgstr "Luo tunnus"
 msgid "User greeting %(user)s"
 msgstr "Hei %(user)s"
 
-#: ckanext/napote_theme/templates/home/layout1.html:7
-msgid "Datasets-page short info"
-msgstr "<b>Palvelukatalogi</b>-sivulla voit tarkastella ja hakea palvelussa julkaistuja rajapintoja."
-
-#: ckanext/napote_theme/templates/home/layout1.html:8
-msgid "Organizations-page short info"
-msgstr "<b>Palveluntuottajat</b>-sivulla on listattuna NAP-palveluun rekisteröityneet palveluntuottajat."
-
-#: ckanext/napote_theme/templates/home/layout1.html:14
-msgid "Heading/NAP home page logged in main heading"
-msgstr "Liikkumispalveluiden tallennus ja julkaisu"
-
-#: ckanext/napote_theme/templates/home/layout1.html:18
-msgid "Transport service short description"
-msgstr "Liikkumispalvelusi olennaisten tietojen kirjaaminen, muokkaaminen ja julkaisu hoituu kätevästi NAP-tietojen hallintapaneelissa."
-
-#: ckanext/napote_theme/templates/home/layout1.html:23
-msgid "Go to data management"
-msgstr "Siirry tietojen hallintapalveluun"
-
-#: ckanext/napote_theme/templates/home/layout1.html:30
-msgid "RAE service short description"
-msgstr "Haluatko tallentaa tai päivittää linja-autoreittejä ja aikataulutietoja RAE-palvelussa?"
-
-#: ckanext/napote_theme/templates/home/layout1.html:38
-msgid "Open RAE-service"
-msgstr "Avaa RAE-palvelu"
+#: ckanext/napote_theme/templates/home/layout1.html:16
+msgid "NAP home page organizations infobox"
+msgstr ""
+"Julkaistaksesi liikkumispalvelutietoja, sinun täytyy liittää henkilökohtainen "
```

Comment:
This suggests that people can join existings orgs at will. The organization admin must add them.